### PR TITLE
docs(installation): Create separate section for each installation platform

### PR DESCRIPTION
**Pull request by astencel-sumo**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -4,28 +4,84 @@ The Sumo Logic OT Distro can be run using either the binary file available in [G
...
 ## Standalone
 
 The Sumo Logic OT Distro is a static Go binary.
 To run it as a standalone process you only need to run the binary file downloaded from
 [Github releases][github_releases] with an appropriate configuration.
 
-1. Set the release version variable:
+Follow the steps for your platform below.
+
+### Linux on Intel
```
Comment (kkujawa-sumo) [inline docs/Installation.md:21]: Maybe it's better to write "Linux on amd64"

Comment (astencel-sumo) [reply docs/Installation.md:21]: Yeah wasn't sure how to phrase it...

Comment (perk-sumo) [reply docs/Installation.md:21]: I'd go with [X86-64 like Wikipedia redirects](https://en.wikipedia.org/wiki/X86-64).
Also to be fair I'd change the build names to that as well.

Comment (perk-sumo) [reply docs/Installation.md:21]: Also it's the same that is shown in the Amazon ECR: 
image

Comment (astencel-sumo) [reply docs/Installation.md]: After discussing this in team, we decided to go with "amd64" and "arm64" for now as these are the names used by upstream and in binary names.

**Review by perk-sumo — APPROVED** [review]

**Review by frankreno — COMMENTED** [review]
LGTM! Will we be able to automate something to keep the links pointing to the latest release every release?

**Comment by sumo-drosiek** [comment]
We should be: https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/linking-to-releases

**Pull request merged**